src/model/channel/reaction.rs

```diff
@@ -3,23 +3,21 @@ use crate::http::CacheHttp;
 use crate::{model::prelude::*};
 use serde::de::{Deserialize, Error as DeError, MapAccess, Visitor};
 use serde::ser::{SerializeMap, Serialize, Serializer};
-use std::{
-    error::Error as StdError,
-    fmt::{
-        Display,
-        Formatter,
-        Result as FmtResult,
-        Write as FmtWrite
-    },
-    str::FromStr
-};
+use std::{error::Error as StdError, fmt::{
+    Display,
+    Formatter,
+    Result as FmtResult,
+    Write as FmtWrite
+}, fmt};
```

Comment:
Unconventional formatting for imports.
```suggestion
use std::{
    error::Error as StdError, 
    fmt::{
        self,
        Display,
        Formatter,
        Result as FmtResult,
        Write as FmtWrite
    },
};
```